Description: Don't use minified JavaScript without source

This partly reverts https://github.com/dask/dask/pull/9306
as we don't have the package they switched to (and given that it's
semi-abandoned, may not want to).

Author: Rebecca N. Palmer <rebecca_palmer@zoho.com>
Forwarded: not-needed

--- a/docs/source/_static/config_converter.js
+++ b/docs/source/_static/config_converter.js
@@ -53,7 +53,7 @@ window.addEventListener('load', (event)
  * @return {object} The parsed data
  */
 function parseYAML(raw) {
-    return YAML.parse(raw)
+    return jsyaml.load(raw)
 }
 
 /**
@@ -151,7 +151,7 @@ function parseCode(raw) {
  * @return {str} YAML configuration
  */
 function dumpYAML(config) {
-    return YAML.stringify(config, 10, 2)
+    return jsyaml.dump(config)
 }
 
 /**
--- a/docs/source/_templates/layout.html
+++ b/docs/source/_templates/layout.html
@@ -1,3 +1,3 @@
 {% extends "!layout.html" %}
 {% set css_files = css_files + ["_static/style.css"] %}
-{% set script_files = script_files + ["_static/yaml.min.js", "_static/config_converter.js"] %}
+{% set script_files = script_files + ["_static/js-yaml.min.js", "_static/config_converter.js"] %}
